hedera-node/configuration/previewnet/settings.txt

```diff
@@ -1,19 +1,25 @@
-checkSignedStateFromDisk, 1
-csvAppend, 0
-csvFileName, MainNetStats
-doUpnp, 0
-enableEventStreaming, true
-eventsLogDir, /opt/hgcapp/eventsStreams
-eventsLogPeriod, 5
-lockLogTimeout, 4000
-maxEventQueueForCons, 1000
-maxOutgoingSyncs, 4
-numConnections, 1000
-showInternalStats, 1
-state.saveStatePeriod, 900
-state.signedStateDisk, 5
-state.signedStateKeep, 100
-throttle7extra, 0.5
-useLoopbackIp, 0
-reconnect.active, false
-reconnect.reconnectWindowSeconds, 300
+checkSignedStateFromDisk,                      1
```

Comment:
May be better to copy previewnet settings to all other settings.txt  to avoid confusion. Other than `hedera-node/configuration/dev/settings.txt` should have `reconnect.active, 1`